### ISSUE TITLE
Add support for EDTF datatype and Local Media

### ISSUE THREAD

**Issue by LeMyst** [LeMyst/WikibaseIntegrator#192]
https://www.mediawiki.org/wiki/Extension:Wikibase_EDTF

https://www.mediawiki.org/wiki/Extension:Wikibase_Local_Media
https://professional.wiki/en/news/wikibase-local-media

**Comment by LeMyst** [comment]
```
                    {
                        "mainsnak": {
                            "snaktype": "value",
                            "property": "P16",
                            "hash": "019e162f8b828560f0ceaad375007d622fae1fe2",
                            "datavalue": {
                                "value": "2019-05-24/2019-05-28",
                                "type": "string"
                            },
                            "datatype": "edtf"
                        },
                        "type": "statement",
                        "qualifiers": {
                            "P17": [
                                {
                                    "snaktype": "value",
                                    "property": "P17",
                                    "hash": "ea90f37fdc7f52223c43a137af87e47a8ce97ee1",
                                    "datavalue": {
                                        "value": "2019-05-24T11:00:00+01:00",
                                        "type": "string"
                                    },
                                    "datatype": "edtf"
                                }
                            ],
                            "P18": [
                                {
                                    "snaktype": "value",
                                    "property": "P18",
                                    "hash": "f97a72476fbc48137333b87ee75c62f56f6a47db",
                                    "datavalue": {
                                        "value": "2019-05-28T23:59:59+01:00",
                                        "type": "string"
                                    },
                                    "datatype": "edtf"
                                }
                            ]
                        },
                        "qualifiers-order": [
                            "P17",
                            "P18"
                        ],
                        "id": "Q4$d294c106-44e1-0cf2-3ba9-b4cb29843eb4",
                        "rank": "normal",
                        "references": [
                            {
                                "hash": "06ed104ca2f7b5dbbd742bf272cdc14b9ca4459c",
                                "snaks": {
                                    "P4": [
                                        {
                                            "snaktype": "value",
                                            "property": "P4",
                                            "hash": "919c4046d941c5e64f188d6d9bd6930b0e2772c3",
                                            "datavalue": {
                                                "value": "https://confuzzled2019.sched.com/",
                                                "type": "string"
                                            },
                                            "datatype": "url"
                                        }
                                    ]
                                },
                                "snaks-order": [
                                    "P4"
                                ]
                            }
                        ]
                    }
```

**Comment by LeMyst** [comment]
Solved.

**Comment by GreenReaper** [comment]
For the curious, Local Media was added in https://github.com/LeMyst/WikibaseIntegrator/commit/d2b70951c36ccc5f0d5a8cb958c4609b5329421c

**Comment by LeMyst** [comment]
Yes, and merged in #143
